src/languageservice/services/yamlCompletion.ts

```diff
@@ -967,11 +967,17 @@ export class YAMLCompletion extends JSONCompletion {
     return this.getInsertTextForValue(value, separatorAfter, type);
   }
 
-  private getLabelForValue(value: string): string {
+  private getLabelForValue(value: unknown): string {
     if (value === null) {
       return 'null'; // return string with 'null' value if schema contains null as possible value
     }
-    return value;
+    if (Array.isArray(value)) {
```

Comment:
```suggestion
    if (Array.isArray(value) || typeof value === 'object') {
```

Does it make sense to do this instead since you are returning the stringified value in both cases anyways